src/css/custom.css

```diff
@@ -0,0 +1,61 @@
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+/* You can override the default Infima variables here. */
+/* You can override the default Infima variables here. */
+:root {
+  --ifm-color-primary: #2a48df;
```

Comment:
We need to decide on a color scheme, I just kept the defaults for now.

Reply:
We have a color scheme in index.html with a link to where I generated it.  That's probably the thing to start with.